.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,68 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ develop ]
+  schedule:
```

Comment:
is there need for a schedule if it's checked on current develop and PR?

Reply:
@karolsojko for this specific branch, you can see results [here](https://github.com/standardnotes/syncing-server-js/security/code-scanning?query=ref%3Arefs%2Fpull%2F57%2Fmerge). I've closed the `Hard-coded credentials` alerts (because it is acceptable in tests files). Alerts are generally available through the `Security` tab > `Code scanning alerts`.

> is there need for a schedule if it's checked on current develop and PR?

Yes. Imagine if there are new queries available from the `security-and-quality` query suite, the analysis needs to run again to get new alerts, so this action needs to running frequently.